Frends.PowerShell.RunCommand/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.0.2] - 2022-12-29
+### Changed
+- Memory leak fix.
```

Comment:
This could be more descriptive. How did you fix the memory leak. And IMO the Changed could be Fixed.